app/exporters/patients_exporter.rb

```diff
@@ -119,6 +119,6 @@ def self.csv_fields(patient)
   end
 
   def self.medications_for(patient)
-    patient.prescription_drugs.flat_map { |drug| [drug.name, drug.dosage] }
+    patient.current_prescription_drugs.flat_map { |drug| [drug.name, drug.dosage] }
```

Comment:
This scope, of course.

Reply:
I know. I think we assumed initially that the `deleted_at` flag was being set on deleted prescription drugs, in which case the default scope would exclude deleted drugs. Now we know this isn't the case.

Reply:
Yeah, maybe during the medication history stuff, we could see if we can move this to be `deleted_at` and not this special `is_deleted` flag.

Reply:
Yup, we've listed it as a "nice-to-do-while-we're-here" 👍